f2/src/components/formik/fileUpload/index.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react'
-import { FormFile, Row } from 'react-bootstrap'
+import { FormFile } from 'react-bootstrap'
 import { UploadButton } from '../button'
 import { acceptableExtensions } from '../../../utils/acceptableFiles'
+import { FormRow } from '../row'
 
 export const FileUpload = ({ field, form, onChange, ...props }) => {
   return (
-    <Row className="form-row">
+    <FormRow>
       <div className="upload-container">
         <FormFile id={props.id} custom>
           <FormFile.Input accept={acceptableExtensions} onChange={onChange} />
           <FormFile.Label />
         </FormFile>
         <UploadButton label={props.label} />
       </div>
-    </Row>
+    </FormRow>
   )
```

Comment:
The original fileUpload component use "VisuallyHidden" of chakra-ui, what component you used to replace that "VisuallyHidden", and does it still have the same functionality(web accessibility to hide content from the visual client, but keep it readable for screen readers)?

Reply:
The new fileUpload component uses FormFile from React Bootstrap - [url](https://react-bootstrap.github.io/components/forms/#forms-custom-file). 

Yes, this interacts with a screen reader. Though the styling needs to be updated to use Emotion.styled instead of CSS.

Reply:
The original VisuallyHidden is to visual hide but reads out texts represented by lingui ID fileUpload.added and fileUpload.removed status. Not sure Bootstrap's FormFile is good enough.

Reply:
The file upload is not what is reading "File added" or "File removed". There is a VisuallyHidden component in the form that is used to read this. I created a styled `<P>` that can be used for this.